$NetBSD: patch-ab,v 1.2 2014/12/10 06:57:25 mef Exp $

Use only _SC_PAGESIZE on DragonFly, _SC_PAGE_SIZE does't exist.

--- heme.c.orig	2005-12-09 14:34:15.000000000 +0000
+++ heme.c
@@ -1332,6 +1332,9 @@ static void init_blocksize()
 	off_len = PAGESIZE;
 # elif defined(_PAGESIZE)
 	off_len = _PAGESIZE;
+# elif defined(__DragonFly__)
+        if((off_len = sysconf(_SC_PAGESIZE)) <= 1)
+		off_len = 4096;
 # else
 	if((off_len = sysconf(_SC_PAGESIZE)) <= 1 &&
 	   (off_len = sysconf(_SC_PAGE_SIZE)) <= 1)
